pkg/apis/apps/types.go

```diff
@@ -157,6 +177,12 @@ type StatefulSetSpec struct {
 	// consists of all revisions not represented by a currently applied
 	// StatefulSetSpec version. The default value is 10.
 	RevisionHistoryLimit *int32
+
+	// PersistentVolumeClaimDeletePolicy indicates the PersistentVolumeClaimDeletePolicy that will
+	// determine when volumes from the VolumeClaimTemplates will be deleted when
+	// the controllering StatefulSet is deleted or scaled down.
```

Comment:
nit: controllering => controlling ?

Reply:
Done.

A controllering resource sounds like it has more fun than a merely controlling resource though ;-)